docs/concepts/scheduled-jobs.md

```diff
@@ -3,6 +3,10 @@
 
 Oftentimes you may want to run a `job` on a regular schedule. For example, fetching from a datasource every morning, compiling an analytics report every month, or detecting model drift every hour.
 
+> Schedules have a minimum interval that will be allowed between two scheduled jobs - by default, a job wouldn't be allowed to be scheduled to run more than 2 times in 10 minutes. 
```

Comment:
Is this correct?
Schedules have a required minimum interval between two scheduled jobs. By default, a job is not allowed to be scheduled twice in a 10 minute period from 00-59 minutes of the hour. 
However, when using schedules like */13 * * * * (every 13th minute), the job is triggered at the 52nd minute and then again at the start of the next hour (minute 0) which means there are 8 minutes between runs. This is allowed.

Reply:
I removed this part `from 00-59 minutes of the hour.` but other than that it makes sense